rc/resources.qrc

```diff
@@ -1,5 +1,6 @@
 <RCC>
     <qresource prefix="/">
         <file>chewing-editor.ico</file>
```

Comment:
Although `chewing-editor.ico` is not included in `resources.qrc`, the ico file is still needed for app icon.
https://doc.qt.io/qt-5/appicon.html#setting-the-application-icon-on-windows